src/core/mapInteractor.js

```diff
@@ -779,10 +826,34 @@ geo.mapInteractor = function (args) {
    */
   ////////////////////////////////////////////////////////////////////////////
   this._handleMouseUp = function (evt) {
+    if (m_clickMaybe) {
+      m_this._handleMouseClick(evt);
+    }
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Handle event when a mouse click is detected.  A mouse click is a simulated
+   * event that occurs when the time between a mouse down and mouse up
+   * is less than the configured duration and (optionally) if no mousemove
+   * events were triggered in the interim.
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this._handleMouseClick = function (evt) {
+
     m_this._getMouseButton(evt);
     m_this._getMouseModifiers(evt);
 
-    // fire a click event here
+    // cancel any ongoing pan action
+    m_this.cancel('pan');
+
+    // unbind temporary handlers on document
+    $(document).off('.geojs');
```

Comment:
`mousedown` adds event handlers to `document` to track the mouse even if it moves outside the map.  This is only needed during an ongoing pan/zoom so the handler is added temporary.  This just ensures that the handlers get unbound when stopping the pan action that had been initiated.